notebooks/scenarios/bigquery/job_helpers.py

```diff
@@ -247,15 +247,15 @@ def create_job_many_columns(user: TestUser) -> TestJob:
     return job
 
 
-def create_job(user: TestUser) -> TestJob:
+def create_random_job(user: TestUser) -> TestJob:
     job_func = random.choice(create_job_functions)
     return job_func(user)
 
 
-def create_jobs(users: list[TestUser], n_per_user: int = 10) -> list[TestJob]:
+def create_jobs(users: list[TestUser], n: int = 10) -> list[TestJob]:
```

Comment:
Should we change the param `n` to `total_jobs` or something else to make it more obvious?